### ISSUE TITLE
Release roadmap

### ISSUE THREAD

**Issue by o1lo01ol1o** [dpkatz/HaskellGBM#2]
Is there a roadmap to release for this package?

**Comment by dpkatz** [comment]
You're right - I need to get this out.  I'll try and find some time to polish it up a bit and get it out onto hackage.

**Comment by o1lo01ol1o** [comment]
ping!
